Lib/test/test_pyrepl/eio_test_script.py

```diff
@@ -0,0 +1,99 @@
+import os
+import sys
+import pty
+import fcntl
+import termios
+import signal
+import errno
+
+
+def handler(sig, f):
+    pass
+
+
+def create_eio_condition():
+    try:
+        # gh-135329: try to create a condition that will actually produce EIO
+        master_fd, slave_fd = pty.openpty()
+        child_pid = os.fork()
+        if child_pid == 0:
+            try:
+                os.setsid()
+                fcntl.ioctl(slave_fd, termios.TIOCSCTTY, 0)
+                p2_pgid = os.getpgrp()
+                grandchild_pid = os.fork()
+                if grandchild_pid == 0:
+                    # Grandchild - set up process group
+                    os.setpgid(0, 0)
+                    # Redirect stdin to slave
+                    os.dup2(slave_fd, 0)
+                    if slave_fd > 2:
+                        os.close(slave_fd)
+                    # Fork great-grandchild for terminal control manipulation
+                    ggc_pid = os.fork()
+                    if ggc_pid == 0:
```

Comment:
```suggestion
                    # Fork grandchild for terminal control manipulation
                    if os.fork() == 0:
```

No need for `ggc_pid` variable.